plugins/content_types/search_result.inc

```diff
@@ -83,10 +83,22 @@ function ting_search_search_result_content_type_render($subtype, $conf, $panel_a
 
     // Set message.
     $message = '';
+
+    $frequency_rank = array(
+      'rank_main_title' => t('Title'),
+      'rank_creator' => t('Creator'),
+      'rank_subject' => t('Subject')
+    );
+    if (isset($search_result) && in_array($search_result->sortUsed, array_keys($frequency_rank))) {
+      $msg = t('Records are sorted by %sortUsed . Select another sort if it is not suited', array('%sortUsed' => $frequency_rank[$search_result->sortUsed]));
+      $message .= '<div class="messages status">' . $msg . '</div>';
```

Comment:
I do not see a reason to wrap the message in two `<div>` tags each with the `message` class.

If the `status` class is the reason for doing this then I suggest that we add that class to the div created on line 101 when needed e.g. by introducing a `$message_classes` variable.

```php
$message = t('Records are sorted by %sortUsed . Select another sort if it is not suited', array('%sortUsed' => $frequency_rank[$search_result->sortUsed]));
$message_classes[] = 'status:
```